8.0-alpine/Dockerfile

```
@@ -33,8 +34,13 @@ RUN apk --no-cache add \
     php8-pecl-memcached \    
     libsodium \
     curl \
+    && echo "date.timezone  = $TZ" > /etc/php8/conf.d/80-siu-timezone.ini \
     && rm -rf /var/cache/apk/*
 
+# https://gitlab.alpinelinux.org/alpine/aports/-/issues/12308
+RUN ln -s php8 /usr/bin/php
```

Se que esta en el path.. pero no seria mejor algo asi para que no quede librado al azar?
```suggestion
RUN ln -s /usr/bin/php8 /usr/bin/php
```

---

Lo tomé como lo tenía la misma gente de alpine :jack_o_lantern: